web/packages/teleport/src/services/websession/websession.ts

```diff
@@ -46,9 +46,10 @@ const session = {
     });
   },
 
-  logoutWithoutSlo(rememberLocation = false) {
+  logoutWithoutSlo(rememberLocation = false, withAccessChangedMessage = false) {
```

Comment:
Same comment about the params, if updating all usages of `logoutWithSlo` is too much, we can take care only of the second argument (and leave a TODO for the first one).